build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1312,6 +1323,57 @@ message OutputSymlink {
   NodeProperties node_properties = 4;
 }
 
+// The termination result of a process, as reported by the operating
+// system to the parent process (e.g., using wait*() on POSIX-like
+// systems, or GetExitCodeProcess() on Microsoft Windows).
+//
+// Consumers SHOULD NOT assume that `reason` is exhaustive. Additional
+// operating system specific methods of process termination may be
+// covered by future versions of this protocol. Consumers MAY assume
+// that any future method indicates unsuccessful process termination.
+message ProcessTerminationResult {
+  message Exited {
+    // The exit code of the process.
+    //
+    // Implementations SHOULD preserve as many bits from the exit code
+    // as possible. On POSIX-like systems, it is recommended to use
+    // waitid(), as siginfo_t's si_status field contains the full exit
+    // code. Functions such as wait() and waitpid() tend to truncate the
+    // exit code to the bottom eight bits.
+    int64 exit_code = 1;
+  }
+
+  message Signaled {
+    // The name of the signal that was delivered that caused the process
+    // to terminate, without the "SIG" prefix (e.g., "BUS", "KILL",
+    // "SEGV").
+    string signal_name = 1;
```

Comment:
Is there a rationale behind choosing a signal name vs. a signal number? (I don't have much of a preference, just curious)